### PR TITLE
Added softbuffer

### PR DESCRIPTION
**Pull request by john01dav**

### CODE REVIEW

File: content/ecosystem/data.toml [hunk 1]
```diff
@@ -1342,4 +1347,4 @@ categories = ["tools"]
 [[items]]
 name = "ziyasal/natura"
 source = "github"
-categories = ["animation"]
+categories = ["animation"]
```
Comment (AngelOnFira) [inline content/ecosystem/data.toml:1350]: Can you add the end-of-file blank line back?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Looks good to me once my comment is resolved :+1:

File: content/ecosystem/data.toml (hunk 1 above)
Comment (john01dav) [reply content/ecosystem/data.toml]: Done

**Comment by john01dav** [comment]
It looks like I made a typo. I'll correct that shortly. Is there any way to build the site locally to make sure that everything is correct in that regard?

File: content/ecosystem/data.toml [hunk 2]
```diff
@@ -1128,6 +1128,11 @@ name = "skulpin"
 source = "crates"
 categories = ["2drendering"]
 
+[[items]
```
Comment (17cupsofcoffee) [inline content/ecosystem/data.toml:1131]:
```suggestion
[[items]]
```

File: content/ecosystem/data.toml [hunk 3]
```diff
@@ -1128,6 +1128,11 @@ name = "skulpin"
 source = "crates"
 categories = ["2drendering"]
 
+[[items]]
+name = ["softbuffer"]
+source = ["crates"]
```
Comment (17cupsofcoffee) [inline content/ecosystem/data.toml:1133]:
```suggestion
name = "softbuffer"
source = "crates"
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**